lib/jazzy/source_declaration/type.rb

```diff
@@ -195,27 +195,27 @@ def ==(other)
 
         # Swift
         'source.lang.swift.decl.function.accessor.address' => {
-          jazzy: 'Address Accessor',
+          jazzy: 'Addressor',
           dash: 'Function',
         }.freeze,
         'source.lang.swift.decl.function.accessor.didset' => {
-          jazzy: 'DidSet Accessor',
+          jazzy: 'didSet Observer',
```

Comment:
Headings should be capitalized even if the corresponding language construct is not. (Consider that the “Class” heading above is capitalized even though lowercase `class` is a language keyword.)

If “DidSet” is bothersome, then this heading should not use a language keyword at all — following the precedent of Enumeration, Structure, and Type Alias.

Reply:
As the description states, all of these changes have been made to align with Apple's developer site, _The Swift Programming Language_, and documentation in the Swift GitHub repository.

In _TSPL_, the usage is always "`didSet` observer" with the keyword in code voice and without changes in capitalization. In these source documents, `didSet` and `willSet` observers are _always_ referred to by their language keyword, while no other entities on our list are _ever_ referred to by their language keyword. I have adhered strictly to precedent and exercised no editorial judgment in this.

Reply:
Before posted that comment, I looked at _The Swift Programming Language_, and was unable to find any examples of `didSet` or `willSet` used in titles or section headings. Can you find an example where they do?

> I have adhered strictly to precedent and exercised no editorial judgment in this.

The rationale in your reply — and this entire thread — _is_ editorial judgement. 😉

Reply:
There are no uses of the term in titles or section headings (of which I'm aware), but you will find that the entity is referred to as the `didSet` observer throughout _TSPL_, and that `didSet` is never spelled in any other fashion even at the beginning of sentences.

I, for one, am entirely comfortable with the notion that the term, in its capitalized form, remains "didSet," like "iPhone" or "pH." Certainly, it is never spelled "DidSet" in Apple or swift.org documentation.

Reply:
Nice catch, @pcantrell, @xwu 

To my eyes, either of the first two rows looks good, but the third looks ugly and unnatural:

| | ...normal case... | Sentence case. | Title/Header Case |
| - | - | - | - |
| ✓ Proper English | ...did‐set observer... | Did‐set observer. | Did‐Set Observer |
| ✓ Rich Text, Code Voice | ...`didSet` observer... | `didSet` observer. | `didSet` Observer |
| ✗ Raw Text, Code Voice | ...didSet observer... | didSet observer. | didSet Observer |

(“DidSet” would be even worse...)

Since the second is what *The Swift Programming Language* does, is it worth trying `<code>didSet</code> Observer` in that field? Does that cause other things to break? How does it look like with Jazzy’s CSS?

The same goes for “will‐set observer”.

Also, should “ivar” be changed to “instance variable” for consistency, since we are speaking English not code?

Reply:
In practice I think this is moot for jazzy right now: users cannot not see these particular strings.

Property observers are not reported by SourceKit as part of document structure so jazzy does not see them and cannot use their type names as markdown or in CLI warning messages.  (Related, they are undocumentable today: try getting Xcode quick help to recognise a doc comment.)

If Swift were to change to include accessors in this view then we would need to make a few changes to the pipeline; if we decided to show them separately in the docs then those would include making the UI and its spelling acceptable to everyone.

The 'Ivar' represents a libclang `CXCursor_ObjCIvarDecl`.  This doesn't sound like something that can occur at the top level of a header file, so probably not a string a user can see in jazzy.  I guess it would be in keeping with the other changes here to change it though.

Reply:
Yes, the point is fairly moot.

However, when it does become possible for this term to be user-facing, the _TSPL_-compliant way to refer to a `didSet` observer in a heading is unambiguously "didSet Observer" without rich text. This is evidenced by existing headings in _TSPL_ such as "The self Property" (note the lack of capitalization and lack of monospaced formatting for the word "self").

I considered and left "ivar" alone for this particular PR, as I could find no evidence of its spelling in Apple documentation. However, "instance variable" seems reasonable.

Reply:
@pcantrell, @SDGGiesbrecht - are you OK with this PR as-is, leaving the strings consistent with Swift and not user-visible?  Or still prefer to leave didSet/willSet unchanged?

(['instance variable' mentions](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/EncapsulatingData/EncapsulatingData.html))

Reply:
@johnfairh Instance variable it is then :)

Reply:
For declaration names that are not user‐visible, everything I have said is irrelevant.

You can even call it `ḓïđŠéț` if you want. ☺